src/layouts/app/connection-mixin.js

```diff
@@ -155,6 +157,13 @@ export default (superClass) =>
         getState()
       );
 
+      getUserLanguage(this.hass).then((language) => {
+        this.hass.selectedLanguage = language;
```

Comment:
Never directly assign, always use `this._updateHass` and only if selectedLanguage is different.